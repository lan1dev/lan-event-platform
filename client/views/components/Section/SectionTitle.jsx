import styled from 'styled-components';
import fonts from '../../constants/fonts';

const SectionTitle = styled.h2`
  font-family: ${fonts.primary};
  font-size: 28px;
`;

export default SectionTitle;
