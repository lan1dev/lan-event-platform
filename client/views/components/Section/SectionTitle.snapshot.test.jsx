import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import SectionTitle from './SectionTitle';

describe('SectionTitle', () => {
  let props;
  let wrapper;

  const Wrapper = () => {
    if (!wrapper) {
      wrapper = shallow(<SectionTitle {...props}>{props.children}</SectionTitle>);
    }
    return wrapper;
  };

  // Reset Component and Props before each test.
  beforeEach(() => {
    props = {};
    wrapper = undefined;
  });

  describe('without props', () => {
    it('Should match snapshot', () => {
      expect(toJson(Wrapper())).toMatchSnapshot();
    });
  });
});
