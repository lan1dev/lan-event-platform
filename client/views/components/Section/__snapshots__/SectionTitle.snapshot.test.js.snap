// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle without props Should match snapshot 1`] = `
.c0 {
  font-family: Poppins;
  font-size: 28px;
}

<h2
  className="c0"
/>
`;
